src/main/scala/codacy/pylint/Pylint.scala

```diff
@@ -10,50 +11,121 @@ import scala.util.{Properties, Success, Try}
 
 object Pylint extends Tool {
 
-  override def apply(path: Path, conf: Option[Seq[PatternDef]], files: Option[Set[Path]])(implicit spec: Spec): Try[Iterable[Result]] = {
-    def isEnabled(issue: Issue) =
-      conf.map(_.exists(_.patternId == issue.patternId)).getOrElse(true)
+   override def apply(path: Path, conf: Option[Seq[PatternDef]], files: Option[Set[Path]])(implicit spec: Spec): Try[Iterable[Result]] = {
+    def isEnabled(issue: Result) = {
+       issue match {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/Codacy/codacy-pylint/file/1737327537/issues/source?bid=2829823&fileBranchId=3054541#l16)